src/js/common/schemaform/reducers/index.js

```diff
@@ -147,35 +153,61 @@ export default function createSchemaFormReducer(formConfig) {
       }
       case SET_SAVE_FORM_STATUS: {
         const newState = _.set('savedStatus', action.status, state);
+        newState.startingOver = false;
+        newState.prefillStatus = PREFILL_STATUSES.notAttempted;
 
         // This is the only time we have a saved datetime
         if (action.status === SAVE_STATUSES.success) {
-          return _.set('lastSavedDate', action.lastSavedDate, newState);
+          newState.lastSavedDate = action.lastSavedDate;
```

Comment:
Right, it doesn't change functionality. I doubt we'll see much of a difference, but it bugs me to do a bunch of these lodash fp functions in a row, since it creates new objects each time.